examples/stackflow-spa/src/seed-design/ui/select-box.tsx

```
@@ -0,0 +1,106 @@
+        <SeedRadioSelectBox.Label>{label}</SeedRadioSelectBox.Label>
+        {description && (
+          <SeedRadioSelectBox.Description>
+            {description}
+          </SeedRadioSelectBox.Description>
+        )}
+      </SeedRadioSelectBox.Content>
+      <SeedRadioSelectBox.Control>
+        <SeedRadioSelectBox.Icon
+          svg={
+            <svg aria-hidden viewBox="0 0 24 24">
+              <circle cx="12" cy="12" r="12" fill="currentColor" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)